makefiles/Base.toml

```diff
@@ -7,26 +7,61 @@ CARGO_PROFILE = "release"
 [tasks.set-image-tag]
 env = { IMAGE_TAG = "${CARGO_MAKE_GIT_HEAD_LAST_COMMIT_HASH}-${CARGO_PROFILE}" }
 
-[env]
-FLUVIO_VERSION = { script = ["cat VERSION"] }
+################################################################################
+# Tasks for building targets matching the build host
+#
 
-[tasks.build-release.mac]
+[tasks.build-dev.mac]
+extend = "build-dev-mac"
+[tasks.build-dev.linux]
+extend = "build-dev-musl"
+
+[tasks.build-prod.mac]
+extend = "build-prod-mac"
+[tasks.build-prod.linux]
+extend = "build-prod-musl"
+
+
+################################################################################
+# Building for x86_64-apple-darwin can only be done on Mac
+#
+
+[tasks.build-dev-mac.mac]
+command = "cargo"
+args = ["build", "--bin", "fluvio", "--target", "x86_64-apple-darwin"]
+
+[tasks.build-prod-mac.mac]
+command = "cargo"
+args = ["build", "--bin", "fluvio", "--release", "--target=x86_64-apple-darwin"]
+
```

Comment:
Split up the `--target` and `x86_64-apple-darwin` to be the same to the `build-dev-mac`